src/pages/Login/index.js

```diff
@@ -2,28 +2,72 @@ import React from 'react'
 import cx from 'classnames'
 import { parse } from 'query-string'
 import { Switch, Route, Link, Redirect } from 'react-router-dom'
-import Icon from '@santiment-network/ui/Icon'
 import Panel from '@santiment-network/ui/Panel/Panel'
 import LoginMetamaskBtn from './LoginMetamaskBtn'
-import LoginEmail from './LoginEmail'
+import LoginEmailForm from './LoginEmailForm'
+import LoginEmailBtn from './LoginEmailBtn'
+import FreeTrialBlock from './FreeTrialBlock'
+import { PATHS } from '../../App'
+import SwipablePages from '../../components/SwipablePages/SwipablePages'
+import MobileWrapper from './Mobile/MobileWrapper'
 import styles from './index.module.scss'
 
-const baseLocation = '/login'
-
-const LoginOptions = () => (
-  <>
-    <h2 className={styles.title}>Log in your account with</h2>
+const LoginDescription = () => (
+  <div className={styles.loginBlock}>
+    <h3 className={styles.title}>Welcome to Santiment</h3>
```

Comment:
@Partysun @GarageInc no?
```suggestion
    <h3 className={styles.title}>Welcome to Sanbase</h3>
```